core/CMakeLists.txt

```diff
@@ -953,12 +953,22 @@ endif (UNIX)
 # Decoding library
 
 # static decoding library
-add_library(drdecode
-  ${DECODER_SRCS}
-  ir/decodelib.c
-  string.c
-  io.c
-  )
+if (HAVE_LIBIPT)
+  add_library(drdecode STATIC
```

Comment:
But it's already static, as the comment at line 955 says: the default for a CMake library is STATIC.   (Plus as an exported library we have to be careful changing anything about it as it can break users -- but this looks like a nop.)

Reply:
Done.